skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -245,20 +251,39 @@ private void addListItemDivider() {
     @Override
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults);
-
         switch (requestCode) {
```

Comment:
And this switch case can be replaced with `if`, because switch is used when 3 or more items are present in the checklist.

Reply:
Sure, will improve soon.